pkgs/applications/graphics/geeqie/default.nix

```diff
@@ -1,23 +1,24 @@
-{ stdenv, fetchurl, pkgconfig, autoconf, automake, gtk2, libpng, exiv2
-, lcms, intltool, gettext, fbida
+{ stdenv, fetchurl, pkgconfig, autoconf, automake, gettext, intltool
+, gtk3, lcms2, exiv2, libchamplain, clutter_gtk, ffmpegthumbnailer, fbida
 }:
 
 stdenv.mkDerivation rec {
   name = "geeqie-${version}";
-  version = "1.3";
+  version = "1.4";
 
   src = fetchurl {
     url = "http://geeqie.org/${name}.tar.xz";
-    sha256 = "0gzc82sy66pbsmq7lnmq4y37zqad1zfwfls3ik3dmfm8s5nmcvsb";
+    sha256 = "0ciygvcxb78pqg59r6p061mkbpvkgv2rv3r79j3kgv3kalb3ln2w";
   };
 
-  preConfigure = "./autogen.sh";
+  # Do not build the changelog as this requires markdown.
```

Comment:
It looks like it i more of because a missing file: https://github.com/BestImageViewer/geeqie/issues/568 If markdown is missing, `README.html` generation is just skipped.

Reply:
You are right. My comment there is wrong. I will remove it when 1.4.1 is released.